h/sentry.py

```diff
@@ -31,6 +31,11 @@ def user_context_data(request):
         'ip_address': request.client_addr,
     }
 
+def extra_context_data(request):
```

Comment:
I'm just unsure of whether adding the extra context in `h/queue.py` suffices or it's needed here as well.